src/helperFunctions/getCloudinaryImageUrl.ts

```diff
@@ -0,0 +1,18 @@
+const cloudinaryImageUrl = `https://res.cloudinary.com/realdevsquad/image/upload`;
+const userBaseUrl = `https://api.realdevsquad.com`;
+
+export const getCloudinaryImgURL = async (userName: string,config:string) => {
+    const memberDataUrl = `${userBaseUrl}/users/${userName}`
+    try {
+        let res = await fetch(memberDataUrl);
+        const data = await res.json()
+        const { user } = data;
+        if (user.picture?.publicId) {
+            const imageUrl = `${cloudinaryImageUrl}/${config}/${user.picture.publicId}`
+            return imageUrl;
```

Comment:
Same comments for the naming convention here too. Please resolve the conversations once you are done with all. Thanks.

Reply:
Thanks. Done